src/sagemaker_containers/_intermediate_output.py

```diff
@@ -61,7 +61,7 @@ def _copy_file(executor, s3_uploader, relative_path, filename):
         dst = os.path.join(tmp_dir_path, relative_path, '{}.{}'.format(_timestamp(), filename))
         shutil.copy2(src, dst)
         executor.submit(_upload_to_s3, s3_uploader, relative_path, dst, filename)
-    except FileNotFoundError:
+    except FileNotFoundError:  # noqa ignore=F821
```

Comment:
I think it's nicer to configure flake8 to understand this is a built-in error: https://github.com/aws/sagemaker-containers/pull/170/files#diff-b91f3d5bd63fcd17221b267e851608e8R20